dora/service_suggestions/models.py

```diff
@@ -184,10 +184,17 @@ def values_to_objects(Model, values):
                     send_suggestion_validated_existing_structure_email(
                         list(emails_contacted), structure, service, contact_email
                     )
-                elif contact_email:
-                    send_suggestion_validated_new_structure_email(
-                        contact_email, structure
-                    )
-                    emails_contacted.add(contact_email)
+                else:
+                    if structure.email:
+                        send_suggestion_validated_new_structure_email(
+                            structure.email, structure
+                        )
+                        emails_contacted.add(structure.email)
+
+                    if contact_email and contact_email not in emails_contacted:
```

Comment:
emails_contacted est vide quand on entre dans le else; donc ce que tu veux vérifier c'est bien juste l'équivalent de
```suggestion
                    if contact_email and contact_email != structure.email:
```
on est d'accord?

Reply:
(auquel cas tu peux laisser ton code tel qu'il est si ça te va, les deux sont équivalents)